README.md

```diff
@@ -38,8 +38,9 @@ provider "aws" {
   region = "us-east-1"
 }
 ```
-### Local variables for RavenDB server resource 
+### Local variables for RavenDB server resource. 
```

Comment:
Variables for RavenDB server Terraform resource parameters ?

Reply:
Variables can be also declared with `.tfvar` rather than `local variables` like in the above.
It's not working with the iterations example so I think it makes more sense to keep it this way.

Reply:
OK let's leave it as "Local variables"